src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,115 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
+import cNames from 'classnames';
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getFilm } from '../../api/api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ onAdd }) => {
+  const [searchError, setSearchError] = useState(false);
+  const [title, setTitile] = useState('');
+  const [movie, setMovie] = useState(null);
+
+  const searchMovie = async() => {
+    const findMovie = await getFilm(title);
+
+    if (findMovie.Response !== 'True') {
+      setSearchError(true);
+      setTitile('');
+      setMovie(null);
+
+      return;
+    }
+
+    const {
+      Title,
+      Plot,
+      Poster,
+      imdbID,
+    } = findMovie;
+
+    const newMovie = {
+      title: Title,
+      description: Plot,
+      imgUrl: Poster,
+      imdbUrl: `https://www.imdb.com/title/${imdbID}`,
+      imdbID,
+    };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+    setMovie(newMovie);
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={cNames('input', {
+                'is-danger': searchError,
+              })}
+              value={title}
+              onChange={({ target }) => {
+                setTitile(target.value);
+                setSearchError(false);
+              }}
+            />
+          </div>
+
+          {searchError && (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
-        <div className="control">
-          <button
-            type="button"
-            className="button is-primary"
-          >
-            Add to the list
-          </button>
+        <div className="field is-grouped">
+          <div className="control">
+            <button
+              type="button"
```

Comment:
if you'll change the type of your button to `submit` you give users opportunity to find a movie with the `enter` key